sim/core/buffs.go

```diff
@@ -226,9 +230,13 @@ func applyBuffEffects(agent Agent, raidBuffs proto.RaidBuffs, partyBuffs proto.P
 	}
 
 	if raidBuffs.IcyTalons {
-		character.PseudoStats.MeleeSpeedMultiplier *= 1.2
+		character.AddStats(stats.Stats{
+			stats.MeleeHaste: 20*HasteRatingPerHastePercent,
```

Comment:
This is also probably incorrect now.

It would be better to revert these for now, and make a separate PR just for them on their own so we can more easily discuss the differences.